README.md

```diff
@@ -127,89 +96,53 @@ To get a local copy up and running, follow these steps.
 
 In order to run this project you need:
 
-<!--
-Example command:
+- In order to run this project you need:
+   - Modern web browser                                                                                                                                                                                                                                                                                                     -           -
 
-```sh
- gem install rails
-```
- -->
 
 ### Setup
 
-Clone this repository to your desired folder:
-
-<!--
-Example commands:
+- Clone this repository to your desired folder with the command `git@github.com:david30-maker/finish-mobile-version.git`
+- cd finish-mobile-version
 
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
 
 ### Install
 
 Install this project with:
 
-<!--
-Example command:
-
-```sh
-  cd my-project
-  gem install
-```
---->
-
+- cd capstone-project
+- Double click on the index.html file
+- 
 ### Usage
 
-To run the project, execute the following command:
-
-<!--
-Example command:
-
-```sh
-  rails server
-```
---->
+- To run the project, execute the following command:
 
+ - Double click on the index.html file
 ### Run tests
 
 To run tests, run the following command:
 
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
+- N/A
 
 ### Deployment
 
 You can deploy this project using:
 
-<!--
-Example:
-
-```sh
-
-```
- -->
+ - Double click on the index.html file
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- AUTHORS -->
+
 
 ## 👥 Authors <a name="authors"></a>
 
 > Mention all of the collaborators of this project.
 
-👤 **Author1**
+👤 **Okpah Igbo**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@githubhandle](https://github.com/david30-maker)
+- Twitter: [@twitterhandle](https://twitter.com/davidigbo) 
+- LinkedIn: [LinkedIn](https://linkedin.com/in/dvid-igbo-464196184/)
 
 👤 **Author2**
```

Comment:
Great job @david30-maker 🤝

- [ ] [Optional] You may remove the second author from your readme file since it is only one person who contributed to this pull request.